### ISSUE TITLE
[BUG] Blocks randomly turning invisible 

### ISSUE THREAD

**Issue by TIKIRobo** [tsunamayo/Starship-EVO#2863]
### Short Description of the issue:
When every I place a block I am unable to see it
...

### Steps to reproduce the issue:
no idea
...

### Environment
- Starship EVO version: 20w36a
- Graphics Card: AMD Radeon RX 5600 XT

### Additional information:
![image](https://user-images.githubusercontent.com/56371294/91901183-b8cbe680-ec54-11ea-89d5-7f54c868fb54.png)
in blue is where I placed some hull that I am currently standing on and the red is were I put some lights but you can see them even though they still produce light. 

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5158568/output_log.txt)

**Comment by TIKIRobo** [comment]
![image](https://user-images.githubusercontent.com/56371294/91901547-3b54a600-ec55-11ea-8255-2463784d0f1c.png)
after a reload ^

**Comment by CurioInventorium** [comment]
I'm getting the same issue after building for a little while. Any new blocks only appear as an outline when looking at them.
It seemed to happen after I deleted a hinge that had some blocks attached to it. I'd also been playing with the player detection blocks.

I had to zip the output.txt log, as it had gone to over 54Mb in size!

[output_log.zip](https://github.com/tsunamayo/Starship-EVO/files/5159236/output_log.zip)
![20200902001519_1](https://user-images.githubusercontent.com/51359034/91915365-41c53b00-ecb2-11ea-97bc-2f32f36a25cc.jpg)

**Comment by CurioInventorium** [comment]
I managed to replicate the situation! It seemed to be triggered because I deleted a hinge block that was in the process of moving.
The set up was a proximity block that switches on a hinge block, that has a few small blocks attached to it.
Triggering the proximity block causes the hinge to swing all the way to one side, I then step out of the proximity blocks range and the hinge starts to return to the off position. While it was moving I deleted the hinge block, which removes anything connected to it.
After this, any further blocks I try to place are invisible. Hope this helps to track down the problem.
